### PR TITLE
[docs] Update readme to link to Github issues instead of Trac

### PR DESCRIPTION
**Pull request by MartijnKaijser**
## Description
Update readme to link to Github issues instead of Trac

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Kodi's developers work hard to make it support a large range of devices and oper
...
 
-* **Coding:** Developers can help Kodi by **[fixing a bug](https://trac.kodi.tv/)**, adding new features, making our technology smaller and faster and making development easier for others. Kodi's codebase consists mainly of C++ with small parts written in a variety of coding languages. Our add-ons mainly consist of python and XML. For more information, please have a look at our **[contributing guide](docs/CONTRIBUTING.md)**.
+* **Coding:** Developers can help Kodi by **[fixing a bug](https://github.com/xbmc/xbmc/issues)**, adding new features, making our technology smaller and faster and making development easier for others. Kodi's codebase consists mainly of C++ with small parts written in a variety of coding languages. Our add-ons mainly consist of python and XML. For more information, please have a look at our **[contributing guide](docs/CONTRIBUTING.md)**.
```
Comment (VelocityRa) [inline README.md]: Consider using a relative link:
```md
(../../issues)
```
so that it works correctly for forks.

Comment (Rechi) [reply README.md]: ../.. isn't always correct

Comment (VelocityRa) [reply README.md]: Oh, when is it not correct? Just curious.

Comment (Rechi) [reply README.md]: e.g. https://github.com/xbmc/xbmc

**Review by hudokkow — APPROVED** [review]

**Pull request merged**